src/pseudopeople/noise.py

```diff
@@ -48,7 +49,7 @@ def noise_form(
     randomness = get_randomness_stream(form.name, seed)
 
     noise_configuration = configuration[form.name]
-    for noise_type in NOISE_TYPES:
+    for noise_type in tqdm(NOISE_TYPES, desc="Applying noise", unit="type"):
```

Comment:
I love tqdm - it's just so easy. But what is `unit="type"` doing?

Reply:
Oh, it is doing xxx type/s. Hmm...

Reply:
maybe `unit="noise types"`?

Reply:
I wanted to keep it short to save on the terminal space. There is already the description of "Applying noise". There should be no ambiguity there...